src/charts/helpers/constants.js

```diff
@@ -86,6 +86,10 @@ export const hoursHuman = [
     '23h',
 ];
 
+export const motion = {
+    duration: 1200,
```

Comment:
How was this value derived?

Reply:
I checked out other libraries to see what they were using, what was looking good in our charts and I ended up with this number.